### ISSUE TITLE
(Minor) Locking plugs interaction

### ISSUE THREAD

**Issue by BreadDain** [IHateMyKite/UnforgivingDevices#47]
Description: locking anal plugs are impossible to struggle from without a chastity key. They also tend to vibrate

Expected: should be possible to lockpick when it has accessibility more than 0, also should act like non-soulgem plugs and not vibrate
Actual: at 90% accessibility there is no possible interaction other than key, and trigger vibrations randomly.

Version of skyrim: SE
Version of mod: 2.0b10
Used patches: DD 5.1, DCL, LBS

**Comment by IHateMyKite** [comment]
Which exact plug are you referring to?

**Comment by BreadDain** [comment]
Locking anal plug

**Comment by IHateMyKite** [comment]
Closing this for now. May open it later if I find time to fix wrong AA
